proposal/aiiuii/README.rst

```diff
@@ -0,0 +1,93 @@
+==============================
+Interbotix Ros Xsarms Toolkits
+==============================
+
+
+
+Basic Information
+=================
+
+The GitHub repository: https://github.com/aiiuii/Interbotix_Ros_Xsarms_Toolkits
+
+Problem to Solve
+================
+
+Recently, my supervising professor purchased a robotic arm for the lab. Unfortunately, no one in the lab had any experience with it, so learning how to use it ended up being one of my professor's assigned tasks.
+
+For this project, I intended to create a GUI system that would allow users to control the robot arm. Although the vendor already had an example, it was too simple in its functionality.
+
+In this project I will use the following tools:
+
+* PyQt: 
+      To create GUI interface.
+* ROS: 
+      The Robot Operating System (ROS) is a set of software libraries and tools that help you build robot applications.
+* IRROS: 
+      The SDK developed by the vendor.It provides API to interact with robotic arm.
+* MoveIt: 
+      MoveIt is ROS's most advanced and flexible library for motion planning and manipulation tasks. It integrates state-of-the-art inverse kinematics solvers, path planning algorithms, and collision detection into a single, unified ROS interface.
+
+
+
+Prospective Users
+=================
+
+Anyone who needs to use robotic arm.
+
+System Architecture
+===================
+
+1. 
+The system will monitor ROS nodes created by IRROS, analyze data and show related information on the screen.
```

Comment:
This reads more like a project for a control system, not a numerical computing system.  A control system requires low latency and in-time response.  A numerical system emphasizes on throughput, scalability, and utilization.

By doing a control system for the term project, it will be hard to practice many of the skills covered in NSD.  For example, the testing techniques differ between control and numeric.

Reply:
I have another idea about implement a simple version of MoveIt cuz it relates to some numerical parts like Motion Planning. This idea may build on OMPL (Open Motion Planning Library). 
Yet, I never learn about that before and it actually has not much relation between my lab work.  😥
If professor like this idea much, I will consider making another proposal. Please let me know if you have any suggestions.
Thank you so much!